app/domain/token_factory.rb

```diff
@@ -10,9 +12,19 @@ class TokenFactory < Dry::Struct
   def signing_key(account)
     slosilo["authn:#{account}".to_sym] || raise(NoSigningKey, account)
   end
-    
+
+  def signed_token_with_expiration(account:, username:)
+    t=Time.now
+    exp_t = t + ENV['CONJUR_AUTHORIZATION_TOKEN_TTL'].to_i * 60
+    signing_key(account).issue_jwt(sub: username, exp: exp_t)
+  end
+
   def signed_token(account:, username:)
-    signing_key(account).issue_jwt(sub: username)
+    if !username.starts_with?('host/') && ENV.include?('CONJUR_AUTHORIZATION_TOKEN_TTL')
```

Comment:
Please add separate configuration for host and users.

Reply:
done